src/inc/corinfo.h

```diff
@@ -276,12 +276,14 @@ enum SystemVClassificationType : unsigned __int8
     SystemVClassificationTypeMemory             = 3,
     SystemVClassificationTypeInteger            = 4,
     SystemVClassificationTypeIntegerReference   = 5,
-    SystemVClassificationTypeSSE                = 6,
+    SystemVClassificationTypeIntegerByReference = 6,
```

Comment:
Minor point - but I think these names confusing. It sounds like it's describing a integer being passed by reference. Also, "byref" is a specific term used in the CLI and so it would be more accurate to use that term. I think it should really be ByRefAsInteger (and the previous one "ReferenceAsInteger"), but at least change "ByReference" to "ByRef".